client/striplighttimer.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import time
+import datetime
+headers={'accept': 'application/json'}
+payload={'power':'true'}
+mario_API="http://mario.clients.shimley:8000/v1/strip/1/power"
+luigi_API="http://luigi.clients.shimley:8000/v1/strip/1/power"
+def timer():
```

Comment:
```suggestion
def timer():
       '''Turns LED strips on and off.'''
```
add a docstring here to explain what the function does.  this will auto-generate documentation if you call `help(timer)`!